server/src/main/java/fr/uga/l3miage/tp2/exo3/models/CommandeEntity.java

```diff
@@ -0,0 +1,21 @@
+package fr.uga.l3miage.tp2.exo3.models;
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.Set;
+
+@Entity
+@Table(name = "Commande")
+public class CommandeEntity {
+    @Id
+    @Column(name = "id")
+    private Long id;
+    @Column(name = "date")
+    private Date date;
+    @Column(name = "montantTotal")
+    private double montantTotal;
+    @ManyToOne
+    private ClientEntity client;
+    @OneToMany
+    private Set<ProduitEntity> produits;
```

Comment:
ici pour le faire de manière unidirectionnelles il te fallait le `@JoinColumn` de cette manière : 
```suggestion
    @OneToMany
    @JoinColumn(name = "id_produit",referencedColumnName = "id")
    private Set< ProduitEntity > produits;
```

car sinon avec ce que tu à écrit, cela créer une table d'association